src/declaration.h

```diff
@@ -534,6 +535,7 @@ class FuncDeclaration : public Declaration
                                         // scopes from having the same name
     VarDeclaration *vthis;              // 'this' parameter (member and nested)
     VarDeclaration *v_arguments;        // '_arguments' parameter
+    Ojbc_FuncDeclaration objc;
```

Comment:
``Ojbc_FuncDeclaratio`` looks like a typo (should it be ``Objc_FuncDeclaration``?)